src/query/functions/tests/it/scalars/testdata/array.txt

```diff
@@ -11,7 +11,7 @@ error:
   --> SQL:1:2
   |
 1 | ['a', 1]
-  |  ^^^ invalid digit found in string while evaluating function `to_uint64('a')` in expr `to_uint64('a')`
+  |  ^^^ invalid digit found in string while evaluating function `to_uint64('a')` in expr `to_uint64('a')`, during run expr: `array(to_uint64('a'), to_uint64(1))`
```

Comment:
`in expr` seems duplicate with `during run expr`, may be we can remove `in expr` here?

Reply:
It's not duplicated, this case is using literal in expr.


see:

```
  |              ^ divided by zero while evaluating function `divide(1, 0)` in expr `1 / expr_a`, during run expr: `if(CAST(cond_a AS Boolean NULL), 1 / expr_a, to_float64(expr_else))`

```